ecommerce/views_test.py

```diff
@@ -131,6 +131,7 @@ def test_order_fulfilled(self):
         assert order.receipt_set.first().data == data
 
         assert send_email.call_count == 0
+        assert post_payment.call_count == 1
```

Comment:
Can you assert that the right order was passed into `post_payment`? You can use `post_payment.call_args[0][0]` to get the order object and make sure its id compares successfully

Reply:
✅